apps/data-types.md

```diff
@@ -168,6 +168,28 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | - | - |  - |
 | name | string | The name of the content page. |
 
+## couponCampaign (create request)
+
+| Attribute | Type | Description |
+| - | - |  - |
+| identifier | string | The alias of the coupon campaign. |
+| name | string | The name of the coupon campaign. |
+| currency | string | The currency code according to ISO 4217. |
+| minimumOrderValue | string | The minimum order value of the coupon campaign, if customers are required to purchase at least a minimum amount to being able to redeem a coupon. |
```

Comment:
Quite long... Maybe shorten it a bit or make it two sentences? Something like this: "The optional minimum order value of the coupon campaign. Customers need to reach this value to redeem a coupon." ?